docs/quickstart_android.md

```diff
@@ -1,6 +1,6 @@
 # Larq Compute Engine: Android Quickstart
 
```

Comment:
The only change I approve in this file is adding the location where the lce_benchmark_model binary is stored after building with bazel. I disagree with rest of the changes.